test/e2e/specs/specs-wpcom/wp-editor__revisions.ts

```diff
@@ -0,0 +1,71 @@
+/**
+ * @group gutenberg
+ */
+
+import {
+	DataHelper,
+	BrowserHelper,
+	TestAccount,
+	GutenbergEditorPage,
+	EditorSettingsSidebarComponent,
+	RevisionsComponent,
+	setupHooks,
+	ParagraphBlock,
+} from '@automattic/calypso-e2e';
+import { Page } from 'playwright';
+
+describe( DataHelper.createSuiteTitle( `Editor: Revisions` ), function () {
+	const accountName = BrowserHelper.targetGutenbergEdge()
+		? 'gutenbergSimpleSiteEdgeUser'
+		: 'simpleSitePersonalPlanUser';
+	let gutenbergEditorPage: GutenbergEditorPage;
+	let editorSettingsSidebarComponent: EditorSettingsSidebarComponent;
+	let page: Page;
+
+	setupHooks( async ( args ) => {
+		page = args.page;
+		gutenbergEditorPage = new GutenbergEditorPage( page );
+
+		const testAccount = new TestAccount( accountName );
+		await testAccount.authenticate( page );
+	} );
+
+	it( 'Go to the new post page', async function () {
+		await gutenbergEditorPage.visit( 'post' );
+	} );
+
+	it( 'Enter post title', async function () {
+		gutenbergEditorPage = new GutenbergEditorPage( page );
+		await gutenbergEditorPage.enterTitle( DataHelper.getRandomPhrase() );
+		await gutenbergEditorPage.saveDraft();
+	} );
+
+	it.each( [ { revision: 1 }, { revision: 2 } ] )(
+		'Create revision $revision',
+		async function ( { revision } ) {
+			const blockHandle = await gutenbergEditorPage.addBlock(
+				ParagraphBlock.blockName,
+				ParagraphBlock.blockEditorSelector
+			);
+			const paragraphBlock = new ParagraphBlock( blockHandle );
+			await paragraphBlock.enterParagraph( `Revision ${ revision }` );
+			await gutenbergEditorPage.saveDraft();
+		}
+	);
```

Comment:
Total nit here: but these revision numbers totally threw me off on first read! Could we perchance make them more accurately reflect their revision number?

The first post title addition and save is technically the first revision. The two revisions added in the next case (that actually modifies the post body) are technically revisions 2 and 3!

I think where it gets confusing is that later you select revision 1, and it isn't actually the content that you labeled as "Revision 1" here

Reply:
I've double checked how revisions modal look like, and I don't see any label similar to `Revision x`:

![image](https://user-images.githubusercontent.com/6549265/150850950-72009f3d-df49-4b40-ba55-5d8d2e32fa7a.png)

Are you seeing something different?

Reply:
Ah, I think I see what you mean.

The confusion was over the creation of `Revision 1` and `Revision 2` when in fact they are meant to be `Revision 2` and `Revision 3` since the original post exists.

Reply:
I think I've fixed this in 34af05384435710ef0a1e8e95a2b4916646084da.

Reply:
Oh yes, sorry, I could have been more clear! I was specifically talking about the text naming of the different revisions. The way you've restructure the test makes this a lot clearer -- I love it!